wiki/Storyboards/pt-br.md

```diff
@@ -0,0 +1,10 @@
+# Storyboards
+
+![Essa cena é mostrada em quanto joga Walkie Talkie Man. Porem é apenas mostrada quando se consegue um hit "Elite Beat 300"](img/Sbpassing.png "Essa cena é mostrada em quanto joga Walkie Talkie Man")
+![Isso é mostrado quando você erra ou faz um hit 50 no combo mostrado anteriormente.](img/Sbfailing.png "Isso é mostrado quando você erra ou faz um hit 50 no combo jogado anteriormente.")
+
+Uma **Storyboard** (conhecida como SB) é uma imagem animada que acompanha o [Beatmap](/wiki/Glossary), normalmente apenas para enfeitar, algumas vezes para mudanças na jogabilidade. Uma storyboard pode ser composta de praticamente qualquer coisa, mas geralmente consiste de efeitos visuais projetados para tornar o mapa mais visualmente agradável e distinto para os usuários. Storyboards são inspiradas em jogos como Osu! Tatakae! Ouendan, onde as storyboards se ajustam a performance do jogador e mostram coisas diferentes dependendo do quão bem o jogador está jogando.
```

Comment:
```suggestion
Uma **Storyboard** (conhecida como SB) é uma imagem animada que acompanha o [Beatmap](/wiki/Glossary), normalmente apenas para enfeitar, algumas vezes para mudanças na jogabilidade. Uma storyboard pode ser composta de praticamente qualquer coisa, mas geralmente consiste de efeitos visuais projetados para tornar o mapa mais visualmente agradável e distinto para os usuários. Storyboards são inspiradas em jogos como o [Osu! Tatakae! Ouendan](/wiki/Glossary#ouendan). Assim como sua inspiração, as storyboards se ajustam a performance do jogador e mostram coisas diferentes dependendo do quão bem o jogador está jogando.
```
Lacks the link to the Ouedan glossary, adjusted a full stop after it- The current one you have is fine too but don't omit the glossary link

Reply:
i didn't omit the link, it was simply not there previously.